worker/online_restore_ee.go

```diff
@@ -62,10 +62,14 @@ func ProcessRestoreRequest(ctx context.Context, req *pb.RestoreRequest) (int, er
 	if err := VerifyBackup(req, &creds, currentGroups); err != nil {
 		return 0, errors.Wrapf(err, "failed to verify backup")
 	}
-
 	if err := FillRestoreCredentials(req.Location, req); err != nil {
 		return 0, errors.Wrapf(err, "cannot fill restore proposal with the right credentials")
 	}
+	restoreId, err := rt.Add()
+	if err != nil {
+		return 0, errors.Wrapf(err, "cannot assign ID to restore operation")
```

Comment:
cannot assign ID to restore operation, because of?

Reply:
wrapf will take care of that. rt.Add() returns multiple kind of errors.  wrapf will combine message to make it more meaningfull. This is the interface it implements. `func (w *withMessage) Error() string { return w.msg + ": " + w.cause.Error() }` Adding something in errors will mess up with the api response. But yes we can add some logging if needed. But seems redundant here.